### ISSUE TITLE
Request for additional example

### ISSUE THREAD

**Issue by kerbstone52** [henrythasler/Leaflet.Geodesic#113]
Hi Henry, I was hoping you might produce an additional example - somewhat of a mid position between the 'Basic Interactive' demo and the 'Interactive measurement' demo. 2 markers, draggable and visible at all times with a geodesic line rendered between. One marker to display a 'tootip' similar to the 'Interactive measurement' demo with details of bearing and distance. Any drag of either marker to update this 'tooltip'. Apologies I'm not a coder, but I've had a go.....quite a long go actually.....and I'm defeated. It seemed easily achievable.

**Comment by henrythasler** [comment]
Yeah, that should not be too hard. I will check it out in the next days.

**Comment by henrythasler** [comment]
Try this demo: [basic-measurement](https://blog.cyclemap.link/Leaflet.Geodesic/basic-measurement.html)

**Comment by kerbstone52** [comment]
Hi Henry, many thanks for doing this. (I actually found the source of my idiocy and managed to do it myself). I implemented it using a compass rose (or actually a protractor) svg as one of the icons. Where this might seem invalid, for my needs I am using the measurements from one point to another visible point on the (viewshed) horizon (only focussed on initial bearing) and the svg overlay is really just a visual aid. A real challenge might be to generate a compass rose svg from the javascript.
![MaceMeasurement](https://github.com/henrythasler/Leaflet.Geodesic/assets/79406264/81eefd98-88ed-4495-95a1-0840ea1dbaad)
